tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -394,10 +407,9 @@ void test_read_abort(void)
 
 	uart_tx(uart_dev, tx_buf, 95, 100 * USEC_PER_MSEC);
 
-	/* Wait for at least one character. RX_RDY event will be generated only
-	 * if there is pending data.
-	 */
-	k_busy_wait(1000);
+	k_timer_start(&read_abort_timer, K_USEC(300), K_NO_WAIT);
+
+	/* RX will be aborted from k_timer timeout */
 
 	uart_rx_disable(uart_dev);
```

Comment:
Wasn't this supposed to be moved to the timeout handler?

Reply:
done